jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/SessionHandler.java

```diff
@@ -66,6 +70,54 @@ public class SessionHandler extends ScopedHandler implements SessionConfig.Mutab
 
     private ContextHandler _contextHandler;
 
+    /**
+     * Wrapper to ensure the correct lifecycle for all sessions in all
+     * contexts to which the request has been dispatched. When a response
+     * is about to be sent back to the client, all of the sessions are
+     * given the opportunity to persist themselves. When the response is
+     * finished being handled, then the sessions have their reference
+     * counts decremented, potentially leading to eviction from their cache,
+     * as appropriate to the configuration of that cache.
+     */
+    private static class SessionStreamWrapper extends HttpStream.Wrapper
```

Comment:
Can you say that this wrapper either replaces or is in addition to the one from the AbstractSessionManager